editor/src/messages/tool/tool_messages/select_tool.rs

```diff
@@ -707,22 +707,27 @@ impl Fsm for SelectToolFsmState {
 					let angle = -mouse_position.angle_to(DVec2::X);
 					let snapped_angle = (angle / snap_resolution).round() * snap_resolution;
 
-					let mut other = graphene_std::Color::from_rgb_str(COLOR_OVERLAY_BLUE.strip_prefix('#').unwrap())
-						.unwrap()
-						.with_alpha(0.25)
-						.to_rgba_hex_srgb();
-					other.insert(0, '#');
-					let other = other.as_str();
 
 					let extension = tool_data.drag_current - tool_data.drag_start;
 					let origin = compass_center - extension;
 					let viewport_diagonal = input.viewport_bounds.size().length();
 
 					let edge = DVec2::from_angle(snapped_angle) * viewport_diagonal;
 					let perp = edge.perp();
+					let edge_dot_horizontal = edge.normalize().dot(DVec2::X).abs();
```

Comment:
This crashes when the user moves their mouse exactly over the starting position in debug mode.

https://github.com/user-attachments/assets/2c5e7a70-778a-457a-a887-471a74775f16